features/topology-map/plugins/org.opennms.features.topology.plugins.topo.history/src/test/java/org/opennms/features/topology/plugins/topo/BundleContextHistoryManagerTest.java

```diff
@@ -258,10 +369,15 @@ private void setBehaviour(GraphContainer graphContainerMock) {
         		.andReturn(selectionManagerMock)
         		.anyTimes();
 
+        Criteria[] criteria = new Criteria[availableCriteria.size()];
+
         EasyMock
                 .expect(graphContainerMock.getCriteria())
-                .andReturn(new Criteria[0])
+                .andReturn(availableCriteria.toArray(criteria))
```

Comment:
Then let me rephrase. In order that the test actually makes sense, we should mock the `addCriteria()`, `removeCriteria()` and `getCriteria()` methods, rather than always return a fixed list. Otherwise the whole purpose of the test is questionable.